### ISSUE TITLE
Part offset in z axis doesn't work.

### ISSUE THREAD

**Issue by DocTrucker** [slic3r/Slic3r#4895]
**Describe the bug**
Part offset in z axis doesn't work.

**To Reproduce**
"File" menu > "Preferences" > Deselect "Auto-centre parts (x,y):" & "Auto-align parts (z=0):"
(See screen shot & ini)
Click ok to clear popup.
Load benchy
Left click to select, right click to raise menu. > "Cut"
Keep upper and lower part.
(Screenshot)
Click perform cut
(Screen shot)
Parts have been moved unexpectedly. (see other bug)
Left click select top part of model.
Right click to raise menu. > "Settings"
Click "#3DBency.stl" in tree window.
Type "24" into "Z:" text entry box in the "Move" section of the popup window.
(Screen shot)
Press ok.
Popup window goes, part hasn't moved as expected.
Ctrl + 8 for front view.
(Screen shot)
Top section still on z=0 plane.
Ctrl + 0 for diagonal view.
Left click select top part of model.
Right click to raise menu. > "Settings"
Click "#3DBency.stl" in tree window.
Type "24" into "Z:" text entry box in the "Move" section of the popup window.
Press enter key.
Top section still on z=0 plane.

**Expected behavior**
Top part should have been offset to z=24 level.

**Screenshots**
See screenshots.

**Desktop (please complete the following information):**
Slic3r Version: 1.3.1-dev
Slic3r Build Commit: 4f5b935
Ubuntu 18.04.

**Additional context**
N/A

[191125_Slic3r_Bug4.zip](https://github.com/slic3r/Slic3r/files/3887016/191125_Slic3r_Bug4.zip)

**Comment by supermerill** [comment]
warning: i can't see the screenshot so maybe i'm wrong.

I think that cut create 2 pieces not 1 piece with 2 parts.
If you save the top part as stl, remove it from slic3r, then go to settings of the bottom part, then add part and select the saved stl, then you should be able to move it.

**Comment by DocTrucker** [comment]
Screenshot should be in the zip file, let me check.

**Comment by supermerill** [comment]
you can drag-drop them

**Comment by DocTrucker** [comment]
I'll bear that in mind for future bug reports/feature requests. Just running through your suggestions.

**Comment by DocTrucker** [comment]
Ok i have removed the previous work flow from cutting. try the following.

**Describe the bug**
Part offset in z axis doesn't work.

**To Reproduce**
"File" menu > "Preferences" > Deselect "Auto-centre parts (x,y):" & "Auto-align parts (z=0):"

![191125_d2_00_Settings](https://user-images.githubusercontent.com/30924036/69550408-dddb4c80-0f92-11ea-94cb-79404e43656f.png)

Click ok to clear popup.
Press "Add" toolbar button, select box, press open.
Press "Add" toolbar button, select box, press open.
Left click to select a box.
Right click for pop-up menu and select settings.
Type "10" into "Z:" text entry box in the "Move" section of the popup window.

![191125_d2_01_OffsetToZ10](https://user-images.githubusercontent.com/30924036/69550443-ed5a9580-0f92-11ea-9d3f-557d73b6fb8a.png)

Press ok.
Ctrl + 8 for front view.

![191125_d2_02_FrontView](https://user-images.githubusercontent.com/30924036/69550478-f8adc100-0f92-11ea-8df1-3a8cd294cf7f.png)

Both boxes still on z=0 plane.

**Expected behaviour**
Top part should have been offset to z=10 level.
Front view should show rectangular object.

**Screenshots**
Screenshots also in zip file.

**Desktop (please complete the following information):**
Slic3r Version: 1.3.1-dev
Slic3r Build Commit: 4f5b935
Ubuntu 18.04.

**Additional context**
N/A

[191125_Slic3r_bug4b.zip](https://github.com/slic3r/Slic3r/files/3887159/191125_Slic3r_bug4b.zip)

**Comment by lordofhyphens** [comment]
Gotta offset relative to something else

**Comment by DocTrucker** [comment]
That doesn't make sense to me. I want to offset relative to world/machine co-ordinates, can you explain or point to how to move a part to z=[a specific value] in the manual?

Why does the preview show it working ok, but then the result fail? What am I doing wrong?

**Comment by DocTrucker** [comment]
Here's more context. I found this bug when trying to resolve the other bug about cutting shifting parts back down to z=0 despite all auto align features being disabled.

Model:
![191125_d3_00_Model](https://user-images.githubusercontent.com/30924036/69557403-71664a80-0f9e-11ea-93aa-babd2fe239ac.png)

Cut window:
![191125_d3_00_Cut](https://user-images.githubusercontent.com/30924036/69557420-7c20df80-0f9e-11ea-96d5-6abd0f6ef85c.png)

Cur result:
![191125_d3_01_CutResult](https://user-images.githubusercontent.com/30924036/69557435-84791a80-0f9e-11ea-9ed1-09c90b044f70.png)

Manual xy alignment (bad, but alignement data has been lost):
![191125_d3_01_PartsAlignedIsh](https://user-images.githubusercontent.com/30924036/69557504-9ce93500-0f9e-11ea-8805-a39b7f74f4be.png)

Attempt to move upper part back up by 2.2 (not as bad as the 2.2 was explicitly specified before, but still extra unneeded workflow):
![191125_d3_03_MoveSpecified](https://user-images.githubusercontent.com/30924036/69557549-b12d3200-0f9e-11ea-86fc-c99ebe18cb42.png)

Result of no move, feet still below base level:
![191125_d3_04_NotMoved](https://user-images.githubusercontent.com/30924036/69557620-d3bf4b00-0f9e-11ea-8a5e-599277337446.png)

**Comment by foreachthing** [comment]
If you want to split parts (i.e. for multi-color printing) you'll need to "add" to an existing part!
1. Perform cut
2. EXPORT model (i.e. TOP cut) as stl (or something else)
3. Double-click the BASE and `Load Part...`
4. Select and ok TOP part
5. Position TOP part in xyz (and maybe the BASE as well.
6. And then it should look like this:
![image](https://user-images.githubusercontent.com/10420187/69583109-64674c80-0fda-11ea-9601-5df548601a15.png)

Sliced:
![image](https://user-images.githubusercontent.com/10420187/69583296-d93a8680-0fda-11ea-9a43-37f98ef8417d.png)

**Comment by DocTrucker** [comment]
Not multicoloured printing, but may as well be. Makes little odds.

Odd workflow. In that case shouldn't the controls to offset the part be hidden when it clearly can't be achieved with a single part?

**Comment by foreachthing** [comment]
@DocTrucker, yes! I, too, find it odd but got used to.
I think it's a good thing, objects drop to the plate. If you miss something and your print starts 0.5 mm in the air ....
You could add a cube (make it really tiny) and add your object to the cube. Then z-offset it as desired.

**Comment by DocTrucker** [comment]
Yeah the work around for offsetting is possible just more work than it needs to be for the users.

It's also not consistant. Load an STL that is floating (my two half box example) and it respects the don't align to z=0 option.

I would hope that I have shown that it would make sense for all opperations to
